Gems/Atom/RHI/Code/Source/RHI/Factory.cpp

```diff
@@ -26,6 +26,7 @@ static bool s_isRenderDocDllLoaded = false;
 #if defined(USE_PIX)
 static AZStd::unique_ptr<AZ::DynamicModuleHandle> s_pixModule;
 static bool s_isPixGpuCaptureDllLoaded = false;
+static bool s_pixGpuMarkersEnabled = false;
```

Comment:
should this still be PixGPUMarkersEnabled if it's for other tools as well? or is gpuMarkersEnabled enough?

Reply:
We don't support this flag with the debug markers in Vulkan/Metal just yet, so will leave as is to not falsely advertise GPU markers on other platforms :)